galsim/config/process.py

```diff
@@ -20,8 +20,20 @@
 import galsim
 import logging
 
-from .input import *
-from .output import *
+
+# Python 2.6 doesn't include OrderdDict natively.  There is a package ordereddict that you
+# can pip install.  But if the user hasn't done that, we'll just read into a regular dict.
+# The only feature that requires the OrderedDict is the truth catalog output.  With a regular
+# dict the columns will appear in arbitrary order.
```

Comment:
Not here I think, since it would usually be irrelevant (if the user isn't outputting a truth catalog), but I added one where the column names are processed in extra_truth.py.  7803620a7b1581a143f2ddf01f81f54308a19f88